src/utils/helpers.js

```diff
@@ -187,12 +187,62 @@ const helpers = {
   },
 
   /**
-   * Map token history object to the expected object in the wallet redux data
+   * @typedef {Object} ReduxTxHistory
+   * @property {string} tx_id
+   * @property {number} timestamp
+   * @property {string} tokenUid
+   * @property {number} balance
+   * @property {boolean} is_voided
+   * @property {number} version
+   * @property {boolean} isAllAuthority
+   */
+
+  /**
+   * @typedef {Object} LibTxHistory
+   * @property {string} txId
+   * @property {number} balance
+   * @property {number} timestamp
+   * @property {boolean} voided
+   * @property {number} version
+   */
+
+  /**
+   * Check if the tx has only inputs and outputs that are authorities
+   *
+   * @param {Object} tx Transaction data
    *
-   * tx {Object} history data element
-   * tokenUid {String} token uid
+   * @return {boolean} If the tx has only authority
    */
-  mapTokenHistory(tx, tokenUid) {
+  isAllAuthority(tx) {
```

Comment:
Could this method become `static`? It doesn't interact with the instance.

Reply:
Well, this is not a class, so there is no `static` methods